### DISCUSSION TITLE
Source of Exclude Older Flag

### DISCUSSION THREAD

**Discussion by kbbr13** [cross-seed/cross-seed#137] [Q&A] [answered]
Hello, thanks a lot for this project. One question, what's the source of this flag? Does it pull from session data and when something was added to rtorrent or something else? I just ran a search with -x 1440 and instead of roughly 100 eligible searches, 1000 searches were conducted... Here is the full string -

npx cross-seed search -u jacket-ip -k jacket-api - t tracker1,tracker2,tracker3 -i /session -s /home-save -e -x 1440

I did add a new tracker to the list, not sure if that could trigger searching from scratch.

**Comment by mmgoodnow** [comment, marked as answer]
It's based on the first time cross-seed encounters the torrents. So as long as you persist your config directory, they won't get searched a second time

**Reply by kbbr13** [reply]
Got it, so adding another tracker would constitute creating a new config. If the trackers searched stayed the same, but exclude timing changed would that also trigger a new config? 

I've been using it now via crontab to search on different intervals and want to control the number of overall searches so search past day regularly and then past 7 days less often, and finally once a month for catch up. 

Would it be possible in the future to include a flag based on when something is added to rtorrent?
